config.rb

```diff
@@ -66,6 +66,12 @@
 # If you want Middleman to listen on a different port, you can set that below
 set :port, 4567
 
+configure :development do
```

Comment:
There's another `configure :development do` defined above on line 42 can these two be combined are should they be separated?